packages/ui-library/src/components/forms/textarea/index.ts

```diff
@@ -154,16 +154,23 @@ export class BlrTextarea extends LitElement {
           ${dynamicStyles}
         </style>
         <div class="${classes} blr-textarea">
-          <div class="label-wrapper">
-            ${BlrFormLabelRenderFunction({
-              labelText: this.label,
-              labelSize: this.size,
-              labelAppendix: this.labelAppendix,
-              forValue: this.textareaId,
-              theme: this.theme,
-              variant: this.hasError ? 'error' : 'label',
-            })}
-          </div>
+        ${
+          this.hasLabel
+            ? html`
+                <div class="label-wrapper">
+                  ${BlrFormLabelRenderFunction({
+                    labelText: this.label,
+                    labelSize: this.size,
+                    labelAppendix: this.labelAppendix,
+                    forValue: this.textareaId,
```

Comment:
Can textareaId be written as camelCase, just like textInputId?

Reply:
Ok I can change it, but isn't the textarea element one word? I know that in storybook we use Text Area to label the component but I'm not sure if that's correct

Reply:
Hmm, yes, seems like it's written like what we have now, so the current style should be fine.

Reply:
Yes, this is exactly how we have treated it in the past, as one word. I remember that I asking the same thing once.